src/components/LibraryDetailPage.tsx

```diff
@@ -119,6 +120,8 @@ export class LibraryDetailPage extends React.Component<LibraryDetailPageProps, L
           fetchLibrary={this.props.fetchLibrary}
         />
         <hr></hr>
+        <PlsIDForm store={this.props.store} current={library.basic_info.pls_id} fetchLibrary={this.props.fetchLibrary} uuid={library.uuid} />
+        <hr></hr>
```

Comment:
Oh, for this I think it’s fine if you just do `<hr />` since it’s a self-closing element, but not a big deal.